site/content/en/docs/manual/advanced/delete_frame.md

```diff
@@ -0,0 +1,50 @@
+---
+title: 'Frame deleting'
+linkTitle: 'Delete frame from task'
+weight: 18
+description: 'This section explains how delete and restore frame from task.'
+---
+
+# Delete frame
+
+You can delete current frame from task. This frame will not be presented either in the UI or in the exported annotation.
+Thus, it is possible to mark corrupted frames that are not subject to annotation.
+
+1. Go to the Job annotation view and than click on the `Delete frame` button.
+
+   ![](/images/image245.jpg)
+
+1. After that you will be asked to confirm frame deleting.
+   **Important note:**  all annotaiton from that frame will be deleted, unsaved annotations
```

Comment:
@ActiveChooN , there is a typo in the sentence but I'm not sure I understand the statement about "unsaved annotations". Could you please explain?

Reply:
We need to flush unsaved annotations before clearing frame annotations and deleting the frame.